src/stakers/stakers.service.ts

```diff
@@ -44,4 +44,47 @@ export class StakersService {
     this.logger.log('Staker data fetched', { withdraw_authority, result });
     return balances;
   }
+
+  async getAllNativeStakeBalances(
+    startDate: string,
+    endDate: string,
+  ): Promise<{ [key: string]: NativeStakeBalanceDto[] } | null> {
```

Comment:
```suggestion
  ): Promise<NativeStakeBalancesDto | null> {
```

Haven't tested it, but shouldn't this work too?